### ISSUE TITLE
Cannot Set `Referer` Header to ChromiumPage Requests

### ISSUE THREAD

**Issue by zarir94** [g1879/DrissionPage#98]
I'm already updated to `DrissionPage==4.0.0b32` which is the latest version. I want to modify headers of http request. All is fine. Almost all headers can be modified (I haven't tested all of them). But the `Referer` Header can't be modified.
This is the example code:
```python
from DrissionPage import ChromiumPage

page = ChromiumPage()
page.set.headers({'Referer': 'https:/google.com', 'Cookie': 'name=value;name2=value2'})
page.get('http://httpbin.org/headers')
```
And the preview is:
![2023-12-28 12_27_29-httpbin org_headers](https://github.com/g1879/DrissionPage/assets/148766170/f3ca2264-9fe5-445d-b8eb-00ac62ac7780)
As you can see, The `Cookie` header is sent but the `Referer` is not. Please fix this in the next update.

**Comment by zarir94** [comment]
Another header that doesn't work is `X-Forwarded-For`

**Comment by juhacz** [comment]
"Accept" header doesn't working too.

**Comment by zarir94** [comment]
It's been 4 months! But the issue is still not fixed! I've rechecked on `DrissionPage==4.0.4.21`

**Comment by g1879** [comment]
嗯，只能设置额外的headers，比如{'abc': '123'}是可以的。
